05/Task05.js

```diff
@@ -1,8 +1,21 @@
 import React from 'react';
+import Form from '../src/components/Form';
+import { Route } from 'react-router-dom';
+import SearchedProducts from '../src/components/SearchedProducts';
+// import NotFound from '../src/components/NotFound';
 
 const Task05 = () => {
     return (
-        <h1>Task05</h1>
+        <>
+            <h1>Task05</h1>
+            <Form />
+            <Route path='/task05/:minPrice?,:maxPrice?-:searchTerm?'>
+                <SearchedProducts />
+            </Route>
+            {/* <Route path='/404.html'>
+                <NotFound />
+            </Route> */}
```

Comment:
Podobnie tutaj. Musisz też pamiętać, że przeglądarka zapisuje sobie przekierowania w pamięci podręcznej więc też możesz mieć wrażenie, że dziwnie to działa ;).
https://zebrahost.com/docs/browsers/clearing_your_https_redirect-cache-from-301-redirects/